app/models/rating.rb

```diff
@@ -1,6 +1,6 @@
 class Rating < ApplicationRecord
   validates_presence_of :rating, :article_id
-  belongs_to :article
+  belongs_to :article, -> { where backyard: false }
```

Comment:
Can you explain?

Reply:
It's scoping down the association to what kind of articles Rating belong to

Reply:
nice! thanks

Reply:
Here this stops backyard articles having ratings.